cpp/msbuild/ice.cpp.props

```diff
@@ -288,11 +266,11 @@
       </PropertyGroup>
 
       <PropertyGroup Condition="'$(IceConfiguration)|$(ConfigurationType)' == 'Debug|StaticLibrary'">
-        <TargetName>$(ProjectName)d</TargetName>
+        <TargetName>$(ProjectName)$(IceSoVersion)d</TargetName>
```

Comment:
This is an incompatible update. Is this a concern? It's not immediately clear to me if/where we use these static libraries? Maybe when building the slice compilers?

If there is any doubt, I'd rather keep the old names and add a TODO to fix it for 3.8.

Reply:
Yes, we use the static libraries for building the compilers, but we do not distribute them.

I will revert and add a 3.8 TODO just to be safe.